packages/upload-client/test/helpers/random.js

```diff
@@ -19,7 +19,7 @@ export async function randomBytes(size) {
     } else {
       crypto.getRandomValues(chunk)
     }
-    size -= bytes.length
+    size -= chunk.length
```

Comment:
FML we were creating random data in tests where the last 65,536 bytes were random but the rest was ZEROs.